test/testdata/configs/config-v4.json

```diff
@@ -15,6 +15,7 @@
     "EnableIncomingMessageFilter": false,
     "EnableMetricReporting": false,
     "EnableOutgoingNetworkMessageFiltering": true,
+    "EnableRequestLogger": false,
```

Comment:
This should be added to config.json.example

Reply:
Yup. It was added to config.json.example. See 3rd file from the top.

Reply:
Yup, sorry, not sure how I missed that!